db/warehouse/migrate/20240113025936_add_ce_assessment_to_form_processor.rb

```diff
@@ -0,0 +1,7 @@
+class AddCeAssessmentToFormProcessor < ActiveRecord::Migration[6.1]
+  def change
+    safety_assured do
+      add_reference :hmis_form_processors, :ce_assessment
```

Comment:
It looks like the warehouse structure file does not reflect this change

Reply:
Thanks! I missed it as I was extracting the diff. Should be fixed now